src/controllers/wallet.js

```diff
@@ -0,0 +1,46 @@
+var WalletModel = require('../models/wallet');
+var Validator = require('validator');
+var Restify = require('restify');
+
+var wallet = {
+    create: function(req, res, next) {
+        // NOTE: request must have content-type set application/json 
+        // or application/x-www-form-urlencoded
+        var error_message = '', msisdn, pin;
+        if (req.params) {
+            msisdn = req.params['msisdn'];
+            pin = req.params['pin'];
+        }
+        
+        if (Object.keys(req.params).length === 0) {
+            error_message = 'Please set content-type header approporiately';
+        }
+        else if (pin === undefined ||
+            msisdn === undefined) {
+            error_message = 'missing argument';
+        }
+        // TODO: this shouldn't just be ZA
```

Comment:
I'm happy with this for now. Mobile phone validation that works globally is hard. Perhaps move the `'en-ZA'` to a configurable thing?